src/common/hooks/useFieldErrorMsg.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { FieldValidator, useField } from 'formik';
+
+type Props = {
+  name: string;
+  inputError?: string;
+  validate?: FieldValidator;
+};
+
+const useFieldErrorMsg = ({ name, inputError, validate }: Props): string | undefined => {
```

Comment:
Am I missing something? Seems like `validate` is not in use here
```suggestion
const useFieldErrorMsg = ({ name, inputError }: Props): string | undefined => {
```

Reply:
validate is passed to useField

Reply:
oops... i overlooked that :) 10x